### PR TITLE
 cw20-merkle-airdrop add possibility to withdraw remaining tokens after expiration

### PR DESCRIPTION
**Pull request by dimiandre**
after expiration owner can withdraw leftover funds for future usage

### CODE REVIEW

**Review by ben2x4 — COMMENTED** [review]
This looks good to me but think we need a config variable to toggle withdrawl on instantiation. Airdrops that want to burn the remaining funds should have withdrawl turned off at instantiation so the owner can't change their mind and actually withdraw the rewards once the airdrop is completed.

**Comment by JakeHartnell** [comment]
Looks good, but just some formatting issues: `cargo fmt`.

File: contracts/cw20-merkle-airdrop/src/contract.rs [hunk 1]
```diff
@@ -284,6 +285,58 @@ pub fn execute_burn(
...
+    // Get balance
+    let balance_to_withdraw = total_amount - claimed_amount;
+
+    // Withdraw the tokens and response
+    let res = Response::new()
+        .add_message(WasmMsg::Execute {
+            contract_addr: cfg.cw20_token_address.to_string(),
+            funds: vec![],
+            msg: to_binary(&Cw20ExecuteMsg::Transfer {
+                recipient: address.clone(),
```
Comment (orkunkl) [inline contracts/cw20-merkle-airdrop/src/contract.rs:326]: best to run `api.addr_validate` on the receipient

File: contracts/cw20-merkle-airdrop/src/msg.rs [hunk 2]
```diff
@@ -35,6 +35,8 @@ pub enum ExecuteMsg {
     },
     /// Burn the remaining tokens after expire time (only owner)
     Burn { stage: u8 },
+    /// Withdraw the remaining tokens after expire time (only owner)
+    Withdraw { stage: u8, address: String}
```
Comment (orkunkl) [inline contracts/cw20-merkle-airdrop/src/msg.rs:39]: Can you run cargo fmt and linter?

**Review by orkunkl — APPROVED** [review]
Added few comments, agree with Ben LGTM.

**Pull request merged**